tasks/dataset-merge/main.py

```diff
@@ -0,0 +1,98 @@
+import logging
+
+from coretex import currentTaskRun, ProjectType, NetworkDataset, CustomDataset, ImageDataset, ImageDatasetClasses
+
+
+ALLOWED_PROJECT_TYPES = [ProjectType.computerVision, ProjectType.other]
+
+
+def mergeCustomDataset(datasets: list[CustomDataset], taskRunId: int, projectId: int) -> CustomDataset:
+    mergeDataset = CustomDataset.createDataset(f"{taskRunId}-merge-custom-dataset", projectId)
```

Comment:
I would rename this to `mergedDataset`, `mergeDataset` is a name for an action like a function, whilst `mergedDataset` is more of a variable name since it represents a result of an action.